druid/src/widget/button.rs

```diff
@@ -52,6 +52,27 @@ impl<T: Data> Button<T> {
         }
     }
 
+    /// Create a new button with the provided label.
+    ///
+    /// Use the `.on_click` method to provide a closure to be called when the
+    /// button is clicked.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use druid::widget::Button;
+    ///
+    /// let button = Button::from_label(Label::new("Increment")).on_click(|_ctx, data: &mut u32, _env| {
```

Comment:
I think including an example is good, but I would prefer the example to demonstrate the value of this method, e.g. I would try to use some `Label` methods that are not available on `Button`.

Reply:
Fair point. I was worried that setting the font would be to much fluff for this but setting the text color seems like a reasonable thing to demonstrate.